x-pack/plugins/maps/public/connected_components/timeslider/timeslider.tsx

```diff
@@ -178,6 +291,31 @@ class KeyedTimeslider extends Component<Props, State> {
             })}
           />
 
+          {this._renderFilteredPeriods()?.length > 0 && (
+            <div className="mapTimeslider__timeRange">
+              <EuiPopover
+                id="metricsPopover"
+                isOpen={this.state.isPopoverOpen}
+                closePopover={this._closePopover}
+                panelPaddingSize="none"
+                ownFocus
+                anchorPosition="upCenter"
+                button={
+                  <EuiButton
+                    iconType="controlsVertical"
```

Comment:
Discussed offline. Consider removing the controlsVertical icon. It's not really necessary, and just taking up space.